frontend/src/assets/manual/en-GB/word-embeddings.md

```diff
@@ -0,0 +1,57 @@
+
+Words that are very similar (i.e. that occur in the same contexts) often have related meanings. You can think of synonyms (like *big* and *large*), but also antonyms (like *black* and *white*). Similarity can also reflect other relations (like *cow* and *calf*; calves are also cows), or simply words that are associated with the same topic (like *cow* and *barn*).
+
+## How are word embeddings created?
+
+The coordinates of each word (and thus how similar words are to each other) are determined by a machine learning model, based on the text of a corpus. Together, they form a model of the words in the corpus, and how those words are used in the text.
+
+This model counts which words tend to occur together in the corpus. (For example, you can count pairs of words that occur at most 4 words apart.) From these counts, the model creates a "map" that tries to capture similarities between words.
+
+You can think of this process like trying to draw up a seating arrangement for a large family at a wedding. You want to make sure that everyone sits next to people they get along with. An observant guest at the event could then use the seating arrangement as a model of the relationships in your family: people who sit next to each other, apparently get along.
+
+There are different algorithms to train word embeddings. What algorithm works, and how exactly this should be configured, can depend on the corpus. If a corpus on I-analyzer has word embeddings available, the information page will describe what algorithm was used.
+
+The result of this algorithm is a set of coordinates for each word. A geographical map is two-dimensional, so the coordinates for each point are two numbers. With word embeddings, word are represented by much longer sequences of numbers, for example 100. This allows them to capture much more complex information than if you tried to arrange all words on a 2D map.
+
+It also means that we cannot easily visualise this map containing all the words. The exact coordinates of a word are to abstract for us to interpret directly. But like with the seating arrangement, if we want to learn something about a word, we need to look at what other words are next to it.
+
+## How can you use word embeddings?
+
+While word embeddings are very abstract representations, they can be a very useful tool. You can evaluate and expand your search terms by exploring similar terms in the model. We discuss a few ways in which you can employ word embeddings in your research process.
+
+### Finding related search terms
+
+When you start your research on a corpus, you often have a few keywords that you are interested in. For example, say that you want to search for *democracy*. A possible pitfall is that an author or speaker may talk about issues of democracy and representation, but not use this specific word. Since those documents will not match your search, you may never realise what you are missing.
```

Comment:
"realise": I think we use the *ize* form in many other places in the software (including its name). Use the same form throughout.